manifests/a/Amazon/AmazonCorretto(x64)/21.0.2.13/Amazon.AmazonCorretto(x64).installer.yaml

```
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: Amazon.AmazonCorretto(x64)

```

```suggestion
PackageIdentifier: Amazon.Corretto.21
```
This manifest should be a new version under the existing identifier